include/MassUpdate.php

```diff
@@ -1332,7 +1332,7 @@ public function doMassUpdateFieldsExistForFocus()
                     case "account_name":
                     case "bool":
                     case "enum":
-					case "dynamicenum":
+		    case "dynamicenum":
```

Comment:
Changes should now be resolved in the following commit. Changed the indentation of the Switch to match PSR-2. The rest of the file could probably do with having the refactor applied to it but I left it alone for now besides the strange indentation issues that were going on with the switch statement.